zerver/openapi/zulip.yaml

```diff
@@ -10074,6 +10113,33 @@ paths:
                               read messages.
 
                               **Changes**: New in Zulip 5.0 (feature level 105).
+                      user_topics:
+                        type: array
+                        description: |
+                          Present is `user_topic` is present in `fetch_event_types`.
+                        items:
+                          type: object
+                          description: |
+                            Object describing the relationship between a user-topic pair.
+                          additionalProperties: false
+                          properties:
+                            stream_id:
+                              type: integer
+                              description: |
+                                The ID of the stream to which the topic belongs.
+                            topic_name:
+                              type: string
+                              description: |
+                                The name of the topic.
+                            last_updated:
+                              type: integer
+                              description: |
+                                An integer UNIX timestamp representing when the user-topic
+                                relationship was changed.
+                            visibility_policy:
+                              type: integer
+                              description: |
+                                The type of visibility policy followed by the user-topic pair.
```

Comment:
Same comment on definitions above, but I guess we don't need to define 0.

@laurynmm may have additional feedback on the API docs here.